### PR TITLE
QS - Public Cloud - First Draft

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10776

This is a pretty rough first draft. Once this is merged, I'll create a PDF to send out to SMEs for feedback/review and we can iterate from there. Suggestions for SMEs gratefully received.

### CODE REVIEW

File: modules/quickstart-public-cloud/pages/clients.adoc [hunk 1]
```diff
@@ -1,4 +1,16 @@
 [[quickstart-publiccloud-clients]]
 = Register Clients
 
-Proactively restore state of the art strategic theme areas via leading-edge models. Distinctively aggregate 2.0 customer service vis-a-vis go forward niche markets. Proactively predominate premier.
+When you have your {productname} Server set up, you are ready to start refgistering clients.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/clients.adoc]:
```suggestion
When you have your {productname} Server set up, you are ready to start registering clients.
```

File: modules/quickstart-public-cloud/pages/clm.adoc [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+This is especially useful if you need to apply updates during a limited maintenance window.
+
+This is achieved through a series of environments that your software channels can move through on their lifecycle.
+Most environment lifecycles include at least test and production environments, but you can have as many environments as you require.
+
+When you have created your project, defined environments, and attached sources and filters, you can build the project for the first time.
+
+For more information about content lifecycle management, see xref:administration:content-lifecycle.adoc[].
+
+When your project is built suiccessfully, you will need to add the new channel to an activation key.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/clm.adoc]:
```suggestion
When your project is built successfully, you will need to add the new channel to an activation key.
```

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 3]
```diff
@@ -1,4 +1,33 @@
 [[quickstart-publiccloud-install]]
 = Installation
 
-Interactively exploit out-of-the-box leadership with collaborative initiatives. Intrinsicly coordinate value-added collaboration and idea-sharing through performance based potentialities. Competently leverage existing technically sound partnerships vis-a-vis highly.
+Public Cloud providers pre-install {productname}, so you do not need to perform any installation steps.
+However, {productname} Server needs to be registered with {scc} to receive updates before you can log in.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc]:
```suggestion
However, {productname} Server needs to be registered with {scc} to receive updates before you can sign in.
```

File: modules/quickstart-public-cloud/pages/clm.adoc [hunk 4]
```diff
@@ -1,4 +1,16 @@
...
-Synergistically facilitate strategic relationships and emerging applications. Dynamically repurpose value-added leadership skills after premium infrastructures. Phosfluorescently build B2C innovation vis-a-vis 24/7 meta-services. Dynamically utilize transparent.
+
+Content lifecycle management allows you to customize and test packages before updating production clients.
+This is especially useful if you need to apply updates during a limited maintenance window.
+
+This is achieved through a series of environments that your software channels can move through on their lifecycle.
+Most environment lifecycles include at least test and production environments, but you can have as many environments as you require.
+
+When you have created your project, defined environments, and attached sources and filters, you can build the project for the first time.
+
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/clm.adoc]:
```suggestion
```
Let's avoid single sentence paras ;)

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 5]
```diff
@@ -1,4 +1,33 @@
...
+
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc]:
```suggestion
```

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 6]
```diff
@@ -1,4 +1,24 @@
...
 = Setting Up
 
-Completely leverage other's superior bandwidth via multimedia based interfaces. Enthusiastically disseminate superior processes with B2B web services. Dynamically brand leveraged leadership and top-line sources. Interactively.
\ No newline at end of file
+
+Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) model.
+This means that you must register instances with the {scc}.
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+
+Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords [package]``suse``, [package]``manager``, [package]``proxy``, or [package]``BYOS``.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/setup.adoc]: These are not \[package\]s :stuck_out_tongue_winking_eye:

File: modules/quickstart-public-cloud/pages/setup.adoc [hunk 7]
```diff
@@ -1,4 +1,24 @@
...
+
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/setup.adoc]:
```suggestion
```

File: modules/quickstart-public-cloud/pages/setup.adoc (hunk 6 above)
Comment (Loquacity) [reply modules/quickstart-public-cloud/pages/setup.adoc]: No, they're not. We'll need to remove the semantic markup from this and the original.

**Pull request merged**